### PR TITLE
Version-getters SDL2/SDL3

### PR DESCRIPTION
**Pull request by Starbuck5**
Continuing my SDL3 patch series.

This takes a look at all "version getters" from SDL and satellite libraries in one go, since they've all consistently updated.

### CODE REVIEW

File: src_c/_pygame.h [hunk 1]
```diff
@@ -106,6 +106,10 @@ PG_UnlockMutex(SDL_mutex *mutex)
 #define PG_SetJoystickEventsEnabled(enabled) \
     SDL_SetJoystickEventsEnabled(enabled)
 
+#define PG_FIND_VNUM_MAJOR(ver) SDL_VERSIONNUM_MAJOR(ver)
+#define PG_FIND_VNUM_MINOR(ver) SDL_VERSIONNUM_MINOR(ver)
+#define PG_FIND_VNUM_MICRO(ver) SDL_VERSIONNUM_MICRO(ver)
```
Comment (ankith26) [inline src_c/_pygame.h:111]: the macros usually copy SDL3 naming, why is this an exception?

Comment (Starbuck5) [reply src_c/_pygame.h:111]: I didn’t want the macro to look like our version number macros, that express our versions.

Comment (Starbuck5) [reply src_c/_pygame.h:111]: Because I don’t want us to be defining SDL symbols or things that look like SDL symbols. We should define our own symbols.

Comment (Starbuck5) [reply src_c/_pygame.h:111]: I've never used PGS_ in any of these SDL3 patches, I was unaware of what that stood for.

I don't want to change the name of all of our compat symbols in this PR.

Comment (ankith26) [reply src_c/_pygame.h:111]: Fair enough then, we can address it later

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR :partying_face:

**Review by MyreMylar — APPROVED** [review]
OK, all looks good to me and the relevant tests and functions for version checking all still work. 👍

**Pull request merged**